Tools/Scripts/webkitpy/layout_tests/controllers/layout_test_finder_legacy.py

```diff
@@ -112,14 +112,21 @@ def find_tests_by_path(self, paths, device_type=None):
                 test_file,
                 is_http_test=self.http_subdir in test_file,
                 is_websocket_test=self.websocket_subdir in test_file or (self.http_subdir in test_file and 'websocket' in test_file),
-                is_wpt_test=(
-                    self.web_platform_test_subdir in test_file
-                    or self.webkit_specific_web_platform_test_subdir in test_file
-                ),
+                is_wpt_test=self._is_wpt_test(test_file),
+                is_wpt_crash_test=self._is_wpt_crash_test(test_file),
             )
             for test_file in self._real_tests(expanded_paths)
         ]
 
+    def _is_wpt_test(self, test_file):
+        return self.web_platform_test_subdir in test_file or self.webkit_specific_web_platform_test_subdir in test_file
+
+    def _is_wpt_crash_test(self, test_file):
+        if not self._is_wpt_test(test_file):
+            return False
+        filename, _ = self._filesystem.splitext(test_file)
+        return filename.endswith('-crash') or self._filesystem.dirname(test_file).endswith(self._port.TEST_PATH_SEPARATOR + 'crashtests')
```

Comment:
It is unclear whether /crashtests/subfolder/test.html should also be considered as a crash test.
Worth adding a test whatever we choose.

Reply:
Actually, I'm gonna make it work instead.